demosplan/DemosPlanCoreBundle/Resources/client/scss/components/_styled-html.scss

```diff
@@ -89,10 +89,20 @@
         min-width: 50px;
 
         > li {
-            margin-bottom: $inuit-base-spacing-unit--small;
+            p {
+                margin: 0;
```

Comment:
Why do you set the p margin to 0, here? Apart from a p inside a li is not valid html, we should only "reset the reset" here (as done with the list style), but not remove spacing that user agents would also apply.

Which brings me to the point that we should check all elements for reasonable spacing within `styled-html`.

Reply:
This type of structure, where the `<p>` tag is inside the `<li>` tag, is created in the editor and passed to the `ListStatement` component as text:
`<ol><li><p>eins</p></li><li><p>zwei</p></li><li><p>drei</p><ul><li><p>punkt</p></li><li><p>punkt</p></li><li><p>punkt</p><ol><li><p>eins</p></li></ol></li></ul></li></ol><p></p>`

The `<p>` tag has a margin-bottom. If I remove the margin only from the `<li>` element, it remains due to the `<p>` tag. That's why I made this adjustment. To resolve this (`<p>` tag is inside the `<li>` tag) , it should be fixed in the editor (DpEditor).

Reply:
remove styling for `<p>` tag inside the `<li>` here https://github.com/demos-europe/demosplan-core/pull/1961/commits/dca72886a18ef5b6488c87f80760270224b3de34

Reply:
when thinking about it for a second time, i guess it is ok to remove the margin on the p element.

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/1961/commits/fd87632472427ec7cd1a10a1cf039224852cf413